typescript/lib/services/mock_service.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020 - present Kurtosis Technologies LLC.
+ * All Rights Reserved.
+ */
+
+const mockServicePort = 1000; 
+
+// Mock service, for testing purposes only
+class MockService {
```

Comment:
Ooooo interesting; this can actually go away now that we've deleted the `Service` interface! Looks like this accidentally got left behind in Golang; I've removed it in https://github.com/kurtosis-tech/kurtosis-client/pull/50